java/impl/aws/src/main/java/co/worklytics/psoxy/aws/ParameterStoreConfigService.java

```diff
@@ -93,6 +93,7 @@ public void putConfigProperty(ConfigProperty property, String value) {
             log.info(String.format("Property: %s, stored version %d", key, parameterResponse.version()));
         } catch (SsmException e) {
             log.log(Level.SEVERE, "Could not store property " + key, e);
+            //q: should we be re-throwing here? caller will be unaware of failure
```

Comment:
yeah, it should! in `ConfigService` putConfigProperty (with retries version) expects an exception to retry!

Reply:
yeah, it's not now